pom.xml

```diff
@@ -97,6 +97,11 @@
         </dependency>
 
         <!-- Main dependencies that need explicit versions -->
+        <dependency>
+            <groupId>com.vladmihalcea</groupId>
+            <artifactId>hibernate-types-55</artifactId>
+            <version>2.14.0</version>
+        </dependency>
```

Comment:
could you explain the purpose of introducing this dependency in PR

Reply:
@mrlumbu I have attempted to explain this in the extended commit message.